static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -393,6 +399,7 @@ define([
                 ['c-tag-links', modules.showMoreTagsLink],
                 ['c-smart-banner', smartAppBanner.init],
                 ['c-adblock', modules.showAdblockMessage],
+                ['c-subscriber-number-form', modules.initSubscriberNumberForm],
```

Comment:
You don't need to create a module method for this. You can just bootstrap with:

```javascript
['c-subscriber-number-form', subscriberNumberForm.init],
```

This is how `techFeedback` and `mediaListener` already work.